elasticsearch/src/test/scala/docs/scaladsl/ElasticsearchSpec.scala

```diff
@@ -1089,6 +1091,54 @@ class ElasticsearchSpec
       assert(readWithSearchParameters.futureValue.toList.sortBy(_.id) == docs.map(_.copy(b = None)))
 
     }
+
+    "be able to use custom routing through searchParams" in assertAllStagesStopped {
```

Comment:
The test just before this is for `be able to use custom searchParams` so it might be worth just extending that test a bit rather than needing a whole new one?

Reply:
Will extend a little bit and add a link to elasticsearch documentation on the routing field